### ISSUE TITLE
Add instructions for tftp booting

### ISSUE THREAD

**Issue by mithro** [skiphansen/panog2_linux#2]
LiteX supports loading the firmware via tftp.

Would be nice to have some instructions to get that working. Should also mean you can support the device with the smaller spi flash.

Can probably steal some instructions from https://github.com/timvideos/litex-buildenv/wiki/HowTo-LCA2018-FPGA-Miniconf-VexRiscv-Renode

**Comment by skiphansen** [comment]
Absolutely.  Will be done before I release any builds for the Rev C device.
